pages/settings/index.js

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import allergens from "../../allergens.json";
+import additives from "../../additives.json";
+import { atomWithStorage, createJSONStorage } from "jotai/utils";
+import Card from "@/components/Card";
+
+const initialAdditives = atomWithStorage("additives", [], {
+  ...createJSONStorage(() => localStorage),
+  delayInit: true,
+});
+
+const initialAllergens = atomWithStorage("allergens", [], {
+  ...createJSONStorage(() => localStorage),
+  delayInit: true,
+});
+
+export default function Settings() {
+  const [showAdditives, setShowAdditives] = useState(true);
```

Comment:
The state itself works fine.
But since you only need it for the toggle purpose, might as well call it toggle.

Reply:
Maybe you are somehow right . But I checked it now in the code and thought about naming it toogle ,setToggle but for me personally it is easier to understand my code and what I am toggling here. Instead I could name it toggleAdditivesOrAllergens but i think thats a way too long . So for me it is ok for now.